bot/seasons/valentines/be_my_valentine.py

```diff
@@ -0,0 +1,236 @@
+import logging
+import random
+import typing
+from json import load
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+from discord.ext.commands.cooldowns import BucketType
+
+from bot.constants import Colours, Lovefest
+
+log = logging.getLogger(__name__)
+
+HEART_EMOJIS = [":heart:", ":gift_heart:", ":revolving_hearts:", ":sparkling_heart:", ":two_hearts:"]
+
+
+class BeMyValentine:
+    """
+    A cog that sends valentines to other users !
+    """
+    id = Lovefest.role_id
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.valentines = self.load_json()
+
+    @staticmethod
+    def load_json():
+        p = Path('bot', 'resources', 'valentines', 'bemyvalentine_valentines.json')
+        with p.open() as json_data:
+            valentines = load(json_data)
+            return valentines
+
+    @commands.group(name="lovefest", invoke_without_command=True)
+    async def lovefest_role(self, ctx):
```

Comment:
This should just invoke the built-in help for the group. You can see an example of this in the AoC Cog:

https://github.com/python-discord/seasonalbot/blob/e2b175bbfdee41bc518f40e430af94e28396966d/bot/seasons/christmas/adventofcode.py#L137-L143

Reply:
got it.